source/kmwbase.js

```diff
@@ -213,7 +213,8 @@ var __BUILD__ = 300;
       var func=util.events[event][i],result=false;
       try { result=func(params); }
       catch(strExcept) { result=false; }//don't know whether to use true or false here      
-      if(!result) { util.currentEvents.pop(); return false; }
+      if(result === false) {return false;}
+	  else {util.currentEvents.pop();}
```

Comment:
Yes, looking at those UI event function callbacks, I see what you mean. Something that should be explicitly addressed in the future. We'll address that in a code review.

Once you can commit the change to your branch I'll merge the PR.